data/en/isvalid.json

```diff
@@ -6,16 +6,16 @@
+		{"name":"type","description":"The valid format for the data.","required":true,"default":"","type":"string","values":["any","array","binary","boolean","cfc","component","class","closure","creditcard","date","datetime", "datetime_object", "email","eurodate","float","function","guid","integer","image","number","numeric","phone","query","range","regex","regular_expression","ssn","social_security_number","string","struct","telephone","time","URL","UUID","usdate","variablename","XML","zip","zipcode"]},
 		{"name":"value","description":"The value to test.","required":true,"default":"","type":"any","values":[]},
-		{"name":"min","description":"The minimum valid value; used only for range validation.","required":true,"default":"","type":"numeric","values":[]},
-		{"name":"max","description":"The maximum valid value; used only for range validation.","required":true,"default":"","type":"numeric","values":[]},
-		{"name":"pattern","description":"A regular expression that the parameter must match;\n used only for regex or regular_expression validation.","required":true,"default":"","type":"string","values":[]}
+		{"name":"min","description":"The minimum valid value; used only for range validation.","required":false,"default":"","type":"numeric","values":[]},
+		{"name":"max","description":"The maximum valid value; used only for range validation.","required":false,"default":"","type":"numeric","values":[]},
+		{"name":"pattern","description":"A regular expression that the parameter must match;\n used only for regex or regular_expression validation.","required":false,"default":"","type":"string","values":[]}
 
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"7", "notes":"CF8+ - added component as a type option.\nCF11+ - no longer allows currency symbols at the start and commas inside a number. Can be reverted to legacy mode by setting this.strictNumberValidation = false in Application.cfc", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-in-k/isvalid.html"},
-		"lucee": {"minimum_version":"", "notes":"", "docs":"https://docs.lucee.org/reference/functions/isvalid.html"},
+		"coldfusion": {"minimum_version":"7", "notes":"CF8+ - added component as a type option.\nCF11+ - no longer allows currency symbols at the start and commas inside a number. Can be reverted to legacy mode by setting this.strictNumberValidation = false in Application.cfc.\nCF2016+ - added datetime_object as a type option.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-in-k/isvalid.html"},
+		"lucee": {"minimum_version":"", "notes":"The types any, cfc, class, closure, float, function, image, number, phone, XML, and zip are Lucee-only.", "docs":"https://docs.lucee.org/reference/functions/isvalid.html"},
```

Comment:
Even though they are missing from Adobe's documentation, `any`, `float`, `function`, and `xml` are usable in ACF.

Reply:
@michaelborn can you update this to take into account @KamasamaK's input?